apps/api/.env.example

```
@@ -9,4 +9,6 @@ FACEBOOK_CLIENT_SECRET=
 NEXT_PUBLIC_DASHBOARD_URL=http://localhost:3003
 NEXT_PUBLIC_ALLOWED_ORIGIN="https://*.vercel.app"
 SUPABASE_URL=
-SUPABASE_SERVICE_ROLE_KEY=
\ No newline at end of file
+SUPABASE_SERVICE_ROLE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_ROLE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)